### PR TITLE
fix for #905 - Prettify query shortcut change

### PR DESCRIPTION
**Pull request by nasehim7**
Keyboard bind edit for Prettify shortcut. Fix for #905

### CODE REVIEW

File: packages/graphiql/src/components/QueryEditor.js [hunk 1]
```diff
@@ -143,6 +143,12 @@ export class QueryEditor extends React.Component {
           }
         },
 
+        'Shift-Ctrl-B': () => {
```
Comment (benjie) [inline packages/graphiql/src/components/QueryEditor.js]: As per https://github.com/graphql/graphiql/issues/905#issuecomment-526861134 we're probably going to move to Monaco at some point; so if `Shift-Alt-F` works in Firefox can we use that instead please? Means less to re-learn in future :)

Also please can you add a comment above this explaining why this change was made?

```suggestion
        /* Shift-Ctrl-P is hard-coded in Firefox for private browsing mode, so add an alternative: */
        'Shift-Alt-F': () => {
```

Comment (nasehim7) [reply packages/graphiql/src/components/QueryEditor.js]: @benjie Yes understandable. I will make the requested changes

**Comment by acao** [comment]
If you cold revise your commit message, we are just about to begin enforcing conventional commits, makes release management easier
`fix(graphiql): prettify keybinding bug for Firefox. Fixes #905`

**Comment by nasehim7** [comment]
@acao Yes Surely I will do that in a moment and update you here

**Comment by nasehim7** [comment]
@acao Done the change to the commit message

**Comment by acao** [comment]
Great, thank you so much!

**Pull request merged**